torchaudio/utils/sox_utils.py

```diff
@@ -1,13 +1,27 @@
 from typing import List, Dict
+from functools import wraps
 
 import torch
 
-from torchaudio._internal import (
-    module_utils as _mod_utils,
-)
 
+def is_sox_available():
+    return torch.ops.torchaudio.is_sox_available()
```

Comment:
For windows support at the moment, the logic has to be `is C++ extension available` AND `is built with sox`

Reply:
so it's `is_module_available('torchaudio._torchaudio') and torch.ops.torchaudio.is_sox_available()`